### PR TITLE
Update 500 template

### PR DESCRIPTION
**Pull request by kwm14**
## Done

- Added Suru hero background
- Split title and content into two sections

## QA

- Pull code
- `./run`
- Visit http://0.0.0.0:8014/500
- See new 500-page template in all its glory

## Issue / Card

Fixes the current style and format.

Screenshot 2019-10-17 at 16 45 55

## Screenshots

Screenshot 2019-10-17 at 16 46 03

### CODE REVIEW

File: 500.html [hunk 1]
```diff
@@ -7,19 +7,18 @@
     lastmod: 2017-08-02T17:20:30+00:00
 ---
 
-<div class="p-strip--light is-deep u-no-padding--bottom">
+<div id="main-content" class="p-strip--image is-dark vfio-hero">
```
Comment (anthonydillon) [inline 500.html]: Please add a background-color to the class as if the image fails then its white text on white background.

File: 500.html [hunk 2]
```diff
@@ -7,19 +7,18 @@
...
-      <div>
-        <h1>500: Internal server error</h1>
-        <p class="p-heading--four">Something went wrong on our end.</p>
-        <p>If you have the time, please help us out by <a class="p-link--external" href="https://github.com/canonical-websites/vanillaframework.io/issues/new">filing a bug.</a></p>
-      </div>
-    </div>
-      <div class="col-6">
-      <img class="b-lazy" src=data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw== data-src="https://assets.ubuntu.com/v1/bcdcf2c8-image-404.svg" alt="">
+    <div class="col 12">
+      <h1 class="p-heading--stylized">500 - Internal server error</h1>
```
Comment (anthonydillon) [inline 500.html]: This class doesn't seem to do anything

**Comment by kwm14** [comment]
Fixed up your comments @anthonydillon 👍

**Comment by anthonydillon** [comment]
Nice but would be better with a `u-no-margin--bottom` class on the h1?

**Pull request merged**